config/routes.rb

```diff
@@ -0,0 +1,11 @@
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show] do
+      resources :comments, only:[:create,:edit,:new,:destroy]
+      resources :likes, only:[:create,:destroy]
+    end
+    end
+  # Defines the root path route ("/")
+  # root "articles#index"
```

Comment:
- [ ] @haftamudesta It's not good practice to have commented code in your codebase, Also would be nice to have a root route in your app.